tests/build-gating.fmf

```diff
@@ -0,0 +1,42 @@
+#
+# Snapshot gating tests for all *LLVM* packages in the mix
+#
+# Compatible with various these distributions:
+#
+#   * Fedora
+#
+
+summary: LLVM Tests for snapshot gating
+discover:
```

Comment:
You could configure here the plan to automatically enable the copr repo you want to test from if you add the following block:
```
prepare:
  how: install
  copr: "@fedora-llvm-team/llvm-snapshots-incubator-$TODAY"
```
And then pass $TODAY from your workflow script like this:
```
tmt <context> run -e TODAY=$today <etc>
```

Reply:
Awesome. Thank you! Done in [a146f40](https://github.com/kwk/llvm-daily-fedora-rpms/pull/74/commits/a146f40b36a5eb0c04d5a46285ec54e1d9af74c7)